assets/css/components/general.css

```diff
@@ -13,10 +13,14 @@
   }
 
   .form-input {
-    @apply rounded box-border border border-solid border-gray-200 w-96 h-10 focus:ring-primary-600;
+    @apply rounded box-border border border-solid border-gray-200 w-96 h-10 focus:ring-primary-600 focus:shadow-inner focus:border focus:border-primary-600 focus:border-solid focus:outline-none;
```

Comment:
```suggestion
    @apply rounded box-border border border-solid border-gray-200 w-96 h-10 focus:ring-primary-600 focus:shadow-inner focus:border-primary-600 focus:outline-none;
```

`focus:border` and `focus:border-solid` are not needed since they are on by default.